src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -26,8 +27,8 @@ public BookController(BookService bookService) {
     }
 
     @GetMapping
```

Comment:
It's ok to leave plain documentation generated by Swagger, but usually it's recommended to add more info for you endpoints. So, add some additional details using `Operation` and `Tag` annotations.